data/korath/korath missions.txt

```diff
@@ -257,3 +257,28 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		random < 5
+		or
+			"reputation: Korath" < -5000
+			"reputation: Korath (Civilian)" < -30
+		not "Korath Boarding Response: active"
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. You had best finish your business with this ship before yours is reduced to a ball of superheated plasma."
+				accept
+	destination "Earth"
```

Comment:
What's the purpose of the "Earth" destination? Just to have somewhere it might complete to clear it out? If this is the case, then having it set to anywhere within 100 jumps of Earth would work better.

Reply:
All the boarding missions with no story destination in the data/human/ directory specified a destination. I suspect it is mandatory, but I am uncertain.

Reply:
I have moved the placeholder destination to Kessel Sepret, which is the main Korath Exile shipyard.

Reply:
from the wiki:

> For missions offered by a ship, you must always specify a destination, even if the mission is designed to always be "declined" - e.g. a message where the pilot thanks you for repairing them but asks for no further help.

So you are correct, for this sort of mission it would be required.